Cargo.toml

```diff
@@ -31,8 +31,8 @@ sentry-tracing = { version = "0.27", optional = true }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_with = { version = "1.12", optional = true }
-solana-client = { version = "1.10", optional = true }
-solana-sdk = { version = "1.10", optional = true }
+solana-client = { version = "1.14", optional = true }
```

Comment:
I would suggest bump crate version - The changing of solana version can break dependent crates